lib/std/fmt.zig

```diff
@@ -543,6 +545,12 @@ pub fn formatIntValue(
         } else {
             @compileError("Cannot print integer that is larger than 8 bits as a ascii");
         }
+    } else if (comptime std.mem.eql(u8, fmt, "u")) {
+        if (@typeInfo(@TypeOf(int_value)).Int.bits <= 21) {
+            return formatUnicodeCodepoint(@as(u21, int_value), options, writer);
+        } else {
+            @compileError("Cannot print integer that is larger than 32 bits as an UTF-8 sequence");
```

Comment:
This error message should contain "21 bits", not "32 bits"

Reply:
oops, I've accidentally copy-pasted the message from the `@compileError` above